docs/contributing.md

```diff
@@ -64,6 +64,17 @@ Create a virtualenv (we name it `ideascube` here but the name is up to you):
 
     mkvirtualenv ideascube
 
+Install the dbus-python bindings first: (make sure you download the version corresponding to your OS)
+
+    wget http://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.0.tar.gz
+    tar xf dbus-python-1.2.0.tar.gz
+    cd dbus-python-1.2.0
+    ./configure --prefix=$VIRTUAL_ENV
+    make
+    make install
+    cd ..
+    rm -fr dbus-python-1.2.0*
+
```

Comment:
Unfortunately, no. :disappointed: 

Unless pip is capable of installing something that uses the Autotools to build?